studio/pages/account/security.tsx

```diff
@@ -0,0 +1,246 @@
+                  {data.totp.length === 2 && <Badge color="green">2 apps configured</Badge>}
+                </>
+              ) : null}
+            </div>
+          }
+          id="auth-app"
+          className="flex flex-row gap-8"
+        >
+          <ProfileCard />
+        </Accordion.Item>
+      </Accordion>
+    </div>
+  )
+}
+
+Security.getLayout = (page) => (
+  <AccountLayout title="Security" breadcrumbs={[{ key: 'security', label: 'Security' }]}>
+    {page}
+  </AccountLayout>
+)
+
+export default Security
+
+const ProfileCard = () => {
+  const [isAddNewFactorOpen, setIsAddNewFactorOpen] = useState(false)
+  const [factorToBeDeleted, setFactorToBeDeleted] = useState<string | null>(null)
+  const { data, isLoading, isError, isSuccess, error } = useMfaListFactorsQuery()
+
+  return (
+    <>
+      <section className="px-6 pt-4">
+        <div>
+          <span className="text-sm prose color-slate-1000">
+            Generate one-time passwords via authenticator apps like 1Password, Authy, etc. as a
+            second factor to verify your identity when prompted during sign-in.
+          </span>
+        </div>
+        {isLoading && (
+          <div className="py-2">
+            {[1, 2].map((number) => {
+              return <ShimmeringLoader key={number} />
+            })}
+          </div>
+        )}
+        {isError && (
+          <div className="py-4">
+            <AlertError error={error} subject="Failed to retrieve account security information" />
+          </div>
+        )}
+        {isSuccess && (
+          <>
+            {data.totp.length === 1 && (
+              <Alert_Shadcn_ variant="default" className="mt-2">
+                <IconAlertCircle className="h-4 w-4" />
+                <AlertTitle_Shadcn_>
+                  We recommend configuring two authenticator apps across different devices
+                </AlertTitle_Shadcn_>
+                <AlertDescription_Shadcn_ className="flex flex-col gap-3">
+                  The two authenticator apps will serve as a backup for each other.
+                </AlertDescription_Shadcn_>
+              </Alert_Shadcn_>
+            )}
+            <div className="py-2">
+              {data.totp.map((factor) => {
+                return (
+                  <div key={factor.id} className="flex flex-row justify-between py-2">
+                    <div className="flex flex-col">
+                      <span className="text-sm text-white">
+                        Name: {factor.friendly_name ?? 'No name provided'}
+                      </span>
+                      <span className="text-sm text-scale-900">Factor ID: {factor.id}</span>
+                    </div>
+                    <div className="flex items-center gap-2">
+                      <span className="text-sm text-scale-900">
+                        Created at {new Date(factor.updated_at).toLocaleString()}
```

Comment:
use dayjs to format, we should include the timezone `'DD MMM YYYY, HH:mm:ss (ZZ)'`

which on that note - i think we should really standardize the way we format timestamps on the dashboard